source/includes/na/ip_addresses/ip_addresses.md.erb

```diff
@@ -1,6 +1,6 @@
 # IP Addresses
 
-In order to ensure a secure integration to Bambora, you must be able to communicate with one of our fully qualified domains. If your integration uses IP Whitelisting, you will need to add Bambora's IP addresses to your whitelist.
```

Comment:
This all LGTM. Out of curiosity, why are we changing this wording from Whitelisting to allowlisting? Is there a difference between the two? I've never heard of allowlist before heh

Reply:
Good question!  This is part of the whole "make language more inclusive" changing we're trying to do.  Allowlisting doesn't have any racial connotations so therefore is a more inclusive & culturally sensitive choice.